models/Product.py

```diff
@@ -13,7 +14,7 @@
 class Item(EmbeddedDocument):
     """Item Schema"""
 
-    _id = ObjectIdField(primary_key=True)
+    _id = ObjectIdField(default=ObjectId(), primary_key=True)
```

Comment:
Maybe try changing default=ObjectId() to default=ObjectId, else we'll get problems when updating/posting new documents. Since if ObjectId() doesn't change due to fast or consecutive document insert we'll be replacing the last document in the db.
```suggestion
    _id = ObjectIdField(default=ObjectId, primary_key=True)
```